src/components/organisms/UiTabs/UiTabs.stories.ts

```diff
@@ -0,0 +1,337 @@
+      :key="key"
+    >
+      <UiTabsItem
+        :title="item.title"
+        :name="item.name"
+        :button-attrs="item.buttonTabAttrs"
+        :content-attrs="item.contentAttrs"
+        v-bind="{
+          'data-testid': item['data-testid'],
+        }"
+      >
+        <UiText>
+          {{ content[item.name] }}
+        </UiText>
+      </UiTabsItem>
+    </template>
+  </UiTabs>`;
+
+const defaultSlotScript = `import { ref } from 'vue';
+import { UiTabs, UiTabsItem, UiText } from '@infermedica/component-library';
+import UiTabsItem from '@infermedica/component-library';
+
+const modelValue = ref('point');
+const items = [
+  {
+    name: 'search',
+    title: 'Search',
+    'data-testid': 'search',
+    buttonTabAttrs: { 'data-testid': 'search-button' },
+    contentAttrs: { 'data-testid': 'search-content' },
+  },
+  {
+    name: 'point',
+    title: 'Point on the body',
+  },
+  {
+    name: 'drop',
+    title: 'Drop body point',
+  },
+];
+const content = {
+  search: 'Serum uric acid concentration',
+  point: '1. Erythrocyte Sedimentation Rate',
+};
+const modifiers = '';`;
+
+const defaultSlotCode = `<template>
+  ${defaultSlotTemplate}
+</template>
+
+<script setup lang="ts">
+${defaultSlotScript}
+</script>`;
+
+const { argTypes } = useArgTypes(UiTabs);
+
+const meta = {
+  title: 'Organisms/Tabs',
+  component: UiTabs,
+  args: {
+    modelValue: 'point',
+    items: complexItemsData,
+    content: {
+      search: 'Serum uric acid concentration',
+      point: '1. Erythrocyte Sedimentation Rate',
+    },
+    modifiers: [],
+  },
+  argTypes: {
+    ...argTypes,
+    content: contentArgTypes,
+  },
+  parameters: {
+    cssProperties: {
+      '--tabs-padding-block': 'var(--tabs-padding-block-start, 0) var(--tabs-padding-block-end, 0)',
+      '--tabs-padding-inline':
+        'var(--tabs-padding-inline-start, 0) var(--tabs-padding-inline-end, 0)',
+    },
+  },
+  decorators: [
+    withModelValue,
+    () => ({
+      template: `<div class="min-h-30">
+      <story />
+    </div>`,
+    }),
+  ],
+} satisfies TabsMetaType;
+
+export default meta;
+
+export const Basic: TabsStoryType = {
+  render: () => ({
+    components: {
+      UiTabs,
+      UiText,
+    },
+    setup(props, { attrs }) {
+      const {
+        modelValue, content, items, modifiers, ...args
+      } = attrs;
+      return {
+        args,
+        modelValue,
+        items,
+        content,
+        modifiers,
+      };
+    },
+    template: basicTemplate,
+  }),
+};
+
+Basic.argTypes = {
+  ...meta.argTypes,
+  modelValue: { control: false },
+  items: { control: 'object' },
+  content: { control: 'object' },
+  modifiers: {
+    ...meta.argTypes.modifiers,
+    name: 'class',
+    description: 'Use this control to add modifier to class.',
+    table: { category: 'html attributes' },
+    control: 'multi-select',
+    options: [
+      'ui-tabs--fixed',
+      'none',
+    ],
+    defaultValue: '',
+  },
+};
```

Comment:
```suggestion
```
modifiers should be handled by `useArgs` probably here we have some issue to solve;
We should use `Basic.argTypes` only when story is different than others. common argTypes should be setup in meta